front/tests/002-project-creation.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { test, expect } from '@playwright/test';
+import { PlaywrightHomePage } from './home-page-model';
+import project from './assets/operationStudies/project.json';
+import { ProjectPage } from './pages/project-page-model';
+import PlaywrightCommonPage from './pages/common-page-model';
+
+test.describe('Test is operationnal study  : project workflow is working properly', () => {
+  test('Create a new project', async ({ page }) => {
+    const playwrightHomePage = new PlaywrightHomePage(page);
+    const projectPage = new ProjectPage(page);
+    const commonPage = new PlaywrightCommonPage(page);
+
+    await playwrightHomePage.goToHomePage();
+    await playwrightHomePage.goToOperationalStudiesPage();
+    await expect(projectPage.getAddProjectBtn).toBeVisible();
+    await projectPage.openProjectModalCreation();
+    await projectPage.setProjectName(project.name);
+    await projectPage.setProjectDescription(project.description);
+
+    await projectPage.setProjectObjectives(project.objectives);
+
+    await projectPage.setProjectFunder(project.funders);
+
+    await projectPage.setProjectBudget(project.budget);
+
+    await commonPage.setTag(project.tags[0]);
```

Comment:
Update the function to set all the tags at once (with a map loop in setTags :))
```suggestion
    await commonPage.setTags(project.tags);
```